app/src/renderer/vuex/modules/ledger.js

```diff
@@ -48,24 +48,46 @@ export default () => {
         externals: state.externals
       }
     },
-    /* TODO: Create a function to detect ledger is connected (i.e unlocked with
-      password and on Home app) */
+    async pollLedgerDevice({ state }) {
```

Comment:
this should throw instead of returning an error

Reply:
Fast Failing 🎉

Reply:
updated